src/WCSimTrackingAction.cc

```diff
@@ -60,6 +65,25 @@ void WCSimTrackingAction::PreUserTrackingAction(const G4Track* aTrack)
     }
   else 
     fpTrackingManager->SetStoreTrajectory(false);
+  
+  // Kill nucleus generated after TrackID 1
```

Comment:
Is this for simulating NEUT's particles?

Reply:
Good point. Frankly speaking, I do not know. I will check this part.